configs/components/razor-server.rb

```diff
@@ -0,0 +1,93 @@
+component "razor-server" do |pkg, settings, platform|
```

Comment:
So, it kind of already is in the layout Scott proposed. Do you have any suggestions as to what I should move where?